docs/user-management/manage-permissions.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 4
+title: Manage Permissions
+---
+
+This section explains user permissions and how to change them for users and API keys.
```

Comment:
We might need to split Permissions into a separate section. Eventually we will have serious RBAC, so that might make sense anyway.

It does not seem logical to me to mix API keys and user management, even though the permissions are identical.

Maybe we should get a convo going on Slack, see what Dustin and the rest think. It's a pickle.